fireant/widgets/highcharts.py

```diff
@@ -111,16 +111,14 @@ def transform(self, data_frame, dimensions, references, annotation_frame=None):
         render_group = []
         split_dimension = self.split_dimension
 
-        if split_dimension:
+        if split_dimension and split_dimension.data_type != DataType.date:
             split_dimension_alias = alias_selector(split_dimension.alias)
 
-            categories = self._categories(
-                result_df,
-                dimension_map,
-                split_dimension_alias,
-            )
+            # Categories method cannot be reused here, given the totals label wouldn't be correctly
+            # mapped to the totals value in the split dimension column.
+            values, _ = self._values_and_dimension(result_df, dimension_map, split_dimension_alias)
 
-            for category in categories:
+            for category in values:
```

Comment:
Does it still make sense to have this variable named category?

Reply:
Changed to value.